src/Morphic-Core/KeyboardEvent.class.st

```diff
@@ -107,11 +107,14 @@ KeyboardEvent >> keyValue [
 
 { #category : #keymapping }
 KeyboardEvent >> modifiedCharacter [
-    self flag: #hack.
-    "Hack me.  When Ctrl is pressed, the key ascii value is not right and we have to do something ugly"
-    ^(self hasSpecialCTRLKeyValue and: [ (#(MacOSX Windows) includes: Smalltalk os current platformFamily) ])
-        ifTrue: [ (self keyValue + $a asciiValue - 1) asCharacter ]    
-        ifFalse: [ self keyCharacter ]
+	self flag: #hack.
+	"Hack me.  When Ctrl is pressed, the key ascii value is not right and we have to do something ugly"
+	^(self hasSpecialCTRLKeyValue and: [ (#(MacOSX Windows) includes: Smalltalk os current platformFamily) ])
```

Comment:
`Smalltalk os platformFamily` is enough here. Why the "current" ?

Reply:
meh, I see it was like that before.
still no need :)

Reply:
@estebanlm I don't know if you know, but now you can suggest one line changes in reviews to make it easier to contribute.

![firefox_2019-01-04_10-18-31](https://user-images.githubusercontent.com/9519971/50681029-2f40bd00-100a-11e9-8776-27d8e2603d95.png)

You can use this icon.